services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -36,7 +36,7 @@ export default class ApplyPage extends React.Component<Props> {
       >
         <Checkbox
           name={`commissionIds.${commission.id}`}
-          value={commission.id.toString()}
+          value={commission.id.toString() && commission.openSeats.toString()}
```

Comment:
`&&` is going to evaluate to the last expression. So now the `value` for the `<input type="checkbox" />`elements is going to be the number of open seats, not the commission ID.

What was your thought process for putting this in?

Reply:
This was supposed to be taken out. This was in the beginning stage where I thought I had to incorporate that.